smoke.test.ts

```diff
@@ -2,232 +2,206 @@ require('selenium-webdriver/chrome');
 require('selenium-webdriver/firefox');
 require('chromedriver')
 
-import { HomePage } from "./pages/home.page";
-import { CartPage } from "./pages/cart.page";
-import { SearchPage } from "./pages/search.page";
-import { WomenPage } from "./pages/women.page";
-import { DressesPage } from "./pages/dresses.page";
-import { TshirtsPage } from "./pages/tshirts.page";
-import { ContactUsPage } from "./pages/contactus.page";
-import { AuthentiPage } from "./pages/authentication.page";
-import { UserAccountPage } from "./pages/user.page";
-import { AddressesPage } from "./pages/addresses.page";
-import { PaymentPage } from "./pages/payment.page";
-
-const { Builder, By } = require('selenium-webdriver');
-const { Driver } = require('selenium-webdriver/chrome');
-const { isAwaitExpression } = require('typescript');
-const rootURL = 'http://automationpractice.com/index.php';
+import { BrowserHelper} from "./browser.helper";
+import { GetPageLocators } from "./page.objects";
 
+const { By } = require('selenium-webdriver');
 
-let driver = new Builder().forBrowser('chrome').build();
+const rootURL = 'http://automationpractice.com/index.php';
 
-let homePage, womenPage, cartPage, searchPage, dressesPage, tshirtsPage, contactUsPage, authenticationPage,
- userAccountPage, addressesPage, paymentPage;
+let browser = new BrowserHelper();
+let pageLocators = new GetPageLocators();
 
 
-describe('Smoke tests for Demo Application',() =>{
+describe('Smoke tests for Demo Application', () => {
 
+  describe('Tests - home page', () => {
 
-  it('Waits for the driver to start', () => {
-    return driver.then((_d: any) => {
-      driver = _d
-    })
-  })
+    it('Check that cart is empty when new user launches the app', async () => {
```

Comment:
as we mentioned before, this test is not a part of smoke test for this app; it should be in the set of regression tests; you can move it to new file, named: regression.test.ts and if you find time to create more test cases that are part of regression, it will be nice to have